tutorial/model/camera/camera.ini

```diff
@@ -0,0 +1,5 @@
+device grabber
+subdevice gazebo_camera
+name /camera
+freq 30
```

Comment:
Good catch, I copied it from the iCub model, I have no idea why it's using `freq` there, since it is not working... Should be fixed now